### DISCUSSION TITLE
Smooth Scroll Conflicting with other JS plugins or JS code

### DISCUSSION THREAD

**Discussion by ghost** [idiotWu/smooth-scrollbar#418] [Q&A]
Hi, a big thanks for this awesome plugin.

I'm a big fan of this Smooth Scrollbar plugin but my bad luck is that I can't use this plugin cause it's conflicting with other js/jQuery plugins. My site just got broken after initializing this plugin. I've tried to remove other plugins one by one, but no luck. I don't know where is this problem conflicting. I'm not a JS expert, that's I can't do anything to use this plugin. Same thing happened to the Locomotive scroll!

Please help me to figure this out.

Thanks.

**Comment by dolphin-wood** [comment]
Can you provide more details, such as error messages or a demo page?

**Reply by ghost** [reply]
Hi,
Thanks for reviewing my post.
Yeah, sure.
Please check the below links.
One is the original demo (Without Smooth Scrollbar JS), another is with Smooth Scrollbar JS.

[Original Demo](https://dev.priyodesign.com/cosec/color-1/)
[Smooth Scrollbar JS Demo](https://demo.priyodesign.com/cosec-smoothscrollbar/)

Please help me if I missed anything to get the result.
Thanks again.

**Reply by dolphin-wood** [reply]
Thanks for the demos! It seems that your site has a lot of elements that interact with scrolling. This plugin is NOT compatible with the native scrolling event, and you have to register scrolling listeners via [`addListener()`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbaraddlistener).

Also, to deal with the sticky header, you can check https://github.com/idiotWu/smooth-scrollbar/discussions/362#discussioncomment-854090.

**Reply by ghost** [reply]
> 

Hi Sir,
Thanks for your quick response. I'll try again. Thanks for your help. You can't imagine how much impressed I'm with this amazing Scrollbar plugin, but my bad luck I can't use this. I'll try again. Thanks.

**Comment by ghost** [comment]
> Thanks for the demos! It seems that your site has a lot of elements that interact with scrolling. This plugin is NOT compatible with the native scrolling event, and you have to register scrolling listeners via [`addListener()`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbaraddlistener).
> 
> Also, to deal with the sticky header, you can check [#362 (comment)](https://github.com/idiotWu/smooth-scrollbar/discussions/362#discussioncomment-854090).

Hi, thanks again for your help. As I said before that I'm not so expert in Javascript, I don't understand how to register scrolling with the addListener() method. Please help me to write the proper <script>...</script>.

Thanks again.
